tests/integration/targets/iosxr_vrf_interfaces/tests/common/_populate_config.yaml

```diff
@@ -0,0 +1,13 @@
+---
+- name: Populate with simple overridden
+  cisco.iosxr.iosxr_vrf_interfaces:
```

Comment:
@KB-perByte should we consider using the `iosxr_config` module here to populate and remove the config? This approach might help us avoid using the `vrf_interfaces` module for setup/teardown in its own tests, ensuring better test isolation and reliability and as we are following this approach in almost all the new resource modules. What do you think?

Reply:
Good point, updated the same.